README.md

```diff
@@ -1,63 +1,200 @@
+
+Go ahead and try this one yourself. Refer back to the previous lesson if you need to.
+
+When you've implemented `tableView(_:cellForRowAtIndexPath:)`, it should look something like this:
+
+```swift
+override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
+    let cell = tableView.dequeueReusableCellWithIdentifier(reuseIdentifier, forIndexPath: indexPath)
+    let songTitle = dirtyDancingSoundtrack[indexPath.row]
+    cell.textLabel?.text = songTitle
+    return cell
+}
+```
+
+What's going on here? First you call `tableView.dequeueReusableCellWithIdentifier(reuseIdentifier, forIndexPath: indexPath)` to get a cell for the given row number. You use the same _reuseIdentifier_ that you specified when setting up the table view in `Main.storyboard` ("DirtyCell"). This part is key—make sure you use the _exact same_ reuse identifier. Otherwise, you'll get a completely different type of cell!
+
+This _may_ be a cell that was already allocated (and is being reused), or it may be a completely new one. On the next line, you get the song title for that row (which is at the same index in the `dirtyDancingSoundtrack` array). On the third line, you set the cell's label to the song title, and then finally return the initialized cell on the fourth line.
+
+Surprisingly, that's it. You're done implementing the table view. The complete implementation of `TableViewController` should look like this:
+
+```swift
+class TableViewController: UITableViewController {
+    let dirtyDancingSoundtrack = [
+        "(I've Had) The Time of My Life",
+        "Be My Baby",
+        "She's Like the Wind",
+        "Hungry Eyes",
+        "STAY",
+        "Yes",
+        "You Don't Own Me",
+        "Hey Baby",
+        "Overload",
+        "Love Is Strange",
+        "Where Are You Tonight?",
+        "In the Still of the Night"
+    ]
+
+    // MARK: - Table View Data Source
+
+    override func numberOfSectionsInTableView(tableView: UITableView) -> Int {
+        return 1
+    }
+
+    override func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return dirtyDancingSoundtrack.count
+    }
+
+    override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCellWithIdentifier("DirtyCell", forIndexPath: indexPath)
+        let songTitle = dirtyDancingSoundtrack[indexPath.row]
+        cell.textLabel?.text = songTitle
+        return cell
+    }
+}
+```
+
+Now you can build and run your app and see it in action!
+
+As you work your way through the next few lessons, you'll build increasingly complex apps backed by table views, and also learn more about using navigation controllers to move easily between different views of your app. But for now, enjoy your brand-new _Dirty Dancing_ app! Nobody puts _you_ in the corner.
```

Comment:
LOL!!!! Oh my god, best end to a reading we might have.

https://www.youtube.com/watch?v=28A9Jgo92GQ